html/index.html

```diff
@@ -0,0 +1,713 @@
+          </div>
+
+          <div class="customers-block____container-info">
+            <div class="customers-block____container-info-title">
+              <h2 class="block-title">Our great customers</h2>
+              <div class="title-underline"></div>
+            </div>
+
+            <img src="/images/customers-block/quotes.png" alt="quotes" />
+            <div>
+              <p>
+                Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium
+                doloremque
+              </p>
+              <p>
+                Laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi
+                architecto beatae vitae dicta sunt explicabo.
+              </p>
+              <p>Stephanie Magion</p>
+            </div>
+
+            <div class="customers-block____container-info-circles">
+              <div class="customers-block____container-info-circles-circle"></div>
+              <div class="customers-block____container-info-circles-circle"></div>
+              <div class="customers-block____container-info-circles-circle blue"></div>
+              <div class="customers-block____container-info-circles-circle"></div>
+              <div class="customers-block____container-info-circles-circle"></div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="started-block">
+        <!-- <img
+          src="/images/started-block/started-background.png"
+          alt="started-background"
+          width="1440"
+          height="406"
+        /> -->
+        <div class="started-block____bg"></div>
+        <div class="started-block____left">
+          <p class="started-block____left-title">IT’S FAST, FREE AND VERY EASY!</p>
+          <h2 class="block-title">Ready to get started?</h2>
+          <div>
+            <p>
+              Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque
+            </p>
+            <p>
+              Laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi
+              architecto beatae vitae dicta sunt explicabo.
+            </p>
+          </div>
+          <a class="started-block____left-apply-btn button" href="#form">Apply Now</a>
+          <button class="started-block____left-request-btn button">Request Service</button>
+        </div>
+        <div class="started-block____right">
+          <div class="started-block____right-title">
+            <svg class="chain-icon" width="24" height="24">
+              <use xlink:href="/images/sprite.svg#chain"></use>
+            </svg>
+            <h2 class="block-title">Quick Link</h2>
+          </div>
+          <div class="started-block____right-box">
+            <div class="started-block____right-box-item">
+              <svg width="24" height="24">
+                <use xlink:href="/images/sprite.svg#pen"></use>
+              </svg>
+              <h4>Tution and Fees</h4>
```

Comment:
This should be a link. This block named "Quick link". Same in blocks under

Reply:
Done.